CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.13.3 / 2015-05-11
+* [BUGFIX] Handle fingerprint collisions properly.
+* [CHANGE] Comments in rules file must start with "#".
+* [ENHANCEMENT] Switch to custom expression language parser.
```

Comment:
"...which generates better error messages and fixes some parsing edge-cases." (so that the value is clear to people)